webui/react/src/components/JupyterLabModal.tsx

```diff
@@ -182,7 +183,7 @@ const JupyterLabModalComponent: React.FC<Props> = ({ workspace }: Props) => {
         showFullConfig
           ? {
               text: 'Read about JupyterLab settings',
-              url: '/docs/reference/api/command-notebook-config.html',
+              url: paths.docs('/architecture/introduction.html#interactive-job-configuration'),
```

Comment:
On localhost, I see the path as **/det**/docs/architecture/introduction.html#interactive-job-configuration when we want the URL to start with `/docs`.  This seems to be because it is made by the Modal / footerLink, instead of a Link component (like the link to docs in the sidebar)

Reply:
fixed